src/components/DailyInfo.tsx

```diff
@@ -119,13 +129,11 @@ function DailyRow({ dayOfWeek }: DailyRowProps): JSX.Element {
 
           <div className="flex justify-center flex-wrap">
             {weaponUpgrades.map((weaponUpgrade, index) => (
-              <div key={index}>
-                <img
+              <div key={index} className="relative h-20 w-20">
```

Comment:
Shouldn't the size be written as props on the Image component ? 

```js
<Image height={} width={} />
```

Reply:
Wouldn't work for responsive (only in px, not in rem)